development/development/coding_guidelines.rst

```diff
@@ -7,6 +7,7 @@ latest versions on area51:
 
 * Rules for `Olympus (3.0.x) code <http://area51.phpbb.com/docs/30x/coding-guidelines.html>`_
 * Rules for `Ascraeus (3.1.x) code <http://area51.phpbb.com/docs/31x/coding-guidelines.html>`_
+* Rules for `Rhea (3.2.x) code <http://area51.phpbb.com/docs/32x/coding-guidelines.html>`_
```

Comment:
This link is not working, it's eventually redirecting to http://area51.phpbb.com/docs/30x/30x/30x/30x/30x/30x/30x/30x/30x/30x/30x/30x/30x/30x/30x/30x/30x/30x/30x/30x/30x/32x/coding-guidelines.html